spin/cmds/meson.py

```diff
@@ -447,6 +492,13 @@ def _get_configured_command(command_name):
     default="html",
     help=f"Format of the gcov report. Can be one of {', '.join(e.value for e in GcovReportFormat)}.",
 )
+@click.option(
```

Comment:
Instead of adding a new flag, what do you think of adding an `html-lcov` option to `--gcov-format`?

Reply:
Yeah, it makes sense to club it. It would need a bit of rework I have used the enum to denote the coverage format directly. Let me see how we can make it neater and more generic.